### DISCUSSION TITLE
How to specify default log level in log.properties file?

### DISCUSSION THREAD

**Discussion by sergeda** [wvlet/airframe#1415] [Q&A] [answered]
I'm looking for a way to specify default log level in log.properties file to use it in combination with Logger.scanLogLevels. Something that would work as Logger.setDefaultLogLevel. Is it possible?

**Comment by xerial** [comment]
I haven't tried that, but using an empty key name (e.g., `"" = debug`) would work to set the root logger level.

**Reply by sergeda** [reply]
@xerial It doesn't work :(

**Reply by xerial** [reply]
Filed a ticket https://github.com/wvlet/airframe/issues/1627

**Reply by xerial** [reply]
@sergeda From the next version, `_root_` logger name can be used to specify the default log level

**Comment by xerial** [comment, marked as answer]
`_root_` logger name can be used for setting the default log level.
